recipes/poppler/all/conanfile.py

```diff
@@ -0,0 +1,265 @@
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import glob
+import os
+
+
+class PopplerConan(ConanFile):
+    name = "poppler"
+    description = "Poppler is a PDF rendering library based on the xpdf-3.0 code base"
+    homepage = "https://poppler.freedesktop.org/"
+    topics = "conan", "poppler", "pdf", "rendering"
+    license = "GPL-2.0-or-later", "GPL-3.0-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = "CMakeLists.txt", "patches/**"
+    generators = "cmake", "cmake_find_package", "pkg_config"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "fontconfiguration": ["generic", "fontconfig", "win32"],
+        "enable_cairo": [True, False],
+        "enable_splash": [True, False],
+        "enable_glib": [True, False],
+        "enable_gobject_introspection": [True, False],
+        "enable_qt": [True, False],
+        "enable_gtk": [True, False],
+        "enable_openjpeg": [True, False],
+        "enable_lcms": [True, False],
+        "enable_libjpeg": [True, False],
+        "enable_png": [True, False],
+        "enable_nss": [True, False],
+        "enable_tiff": [True, False],
+        "enable_dctdecoder": [True, False],
+        "enable_libcurl": [True, False],
+        "enable_zlib": [True, False],
+        "use_float": [True, False],
+
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "fontconfiguration": "generic",
+        "enable_cairo": False,
```

Comment:
Interesting, Does Poppler have an internal Cairo implementation?

Reply:
No, this should require cairo. This is a bug.